docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,95 @@
+# Getting Started
+
+## Requirements
+
+This project includes Front-End and Contracts because of this we expect the current development workspace to have installed;
+
+- [Node.js v16.15.0 or latest stable](https://nodejs.org/en/) we recommend use [nvm](https://github.com/nvm-sh/nvm) to install.
+- [PNPM v7.1.7 or latest stable](https://pnpm.io/installation/)
+- [Rust toolchain v0.16.0 or latest stable](https://www.rust-lang.org/tools/install)
+- [Forc v0.14.5 or latest stable](https://fuellabs.github.io/sway/latest/introduction/installation.html)
+- [Docker v0.8.2 or latest stable](https://docs.docker.com/get-docker/)
+- [Docker Compose v2.6.0 or latest stable](https://docs.docker.com/get-docker/)
+
+## Running project locally
+
+### 📚 - Getting the Repository
+
+1. Visit the [SwaySwap](https://github.com/FuelLabs/swayswap) repo and fork the project.
+2. Then clone your forked copy to your local machine and get to work.
+
+```sh
+git clone https://github.com/FuelLabs/swayswap
+cd swayswap
+```
+
+### 📦 - Install dependencies
+
+```sh
+pnpm install
+```
+
+### 📒 - Run Local Node
+
+In this step, we are going to;
+
+- launch a local `fuel-core` node;
+- launch a local `faucet` API;
+- Setup `swayswap-scripts`;
+- Build and Deploy the `swayswap contracts`.
+
+```sh
+pnpm services:setup
+```
+
+### 💻 - Run Web App
+
+Start the local development `front-end`. After running the command you can open on
+you browser [http://localhost:3000](http://localhost:3000).
+
+```
+pnpm dev
+```
+
+### ✨ - First steps
+
+This is guided step-by-step tutorial to create your first pool and be able to
```

Comment:
I think the correct grammar is "This guided step-by-step tutorial will allow you to create your first pool and play around with the application"